libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/coordinator/CompassPlaceholderBinder.kt

```diff
@@ -0,0 +1,31 @@
+package com.mapbox.navigation.dropin.coordinator
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI
+import com.mapbox.navigation.core.lifecycle.MapboxNavigationObserver
+import com.mapbox.navigation.dropin.NavigationViewContext
+import com.mapbox.navigation.dropin.R
+import com.mapbox.navigation.dropin.component.map.CompassPlaceholderComponent
+import com.mapbox.navigation.dropin.databinding.MapboxCompassPlaceholderLayoutBinding
+import com.mapbox.navigation.ui.base.lifecycle.UIBinder
+
+@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
+internal class CompassPlaceholderBinder(
+    private val context: NavigationViewContext,
+) : UIBinder {
+
+    override fun bind(viewGroup: ViewGroup): MapboxNavigationObserver {
+        val binding = inflateLayout(viewGroup)
+        return CompassPlaceholderComponent(binding.compassPlaceholder, context.styles.mapCompassParams)
+    }
+
+    private fun inflateLayout(viewGroup: ViewGroup): MapboxCompassPlaceholderLayoutBinding {
+        val layoutInflater =
+            viewGroup.context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
+        viewGroup.removeAllViews()
+        layoutInflater.inflate(R.layout.mapbox_compass_placeholder_layout, viewGroup)
```

Comment:
```suggestion
        viewGroup.removeAllViews()
        View.inflate(viewGroup.context, R.layout.mapbox_compass_placeholder_layout, viewGroup)
```
Same for `ScalePlaceholderBinder`.